packages/auto-drive/src/ipld/chunker.ts

```diff
@@ -1,103 +1,138 @@
 import { PBNode } from '@ipld/dag-pb'
+import { BaseBlockstore } from 'blockstore-core'
+import { AwaitIterable } from 'interface-store'
 import { CID } from 'multiformats'
-import { cidOfNode } from '../cid/index.js'
-import { OffchainMetadata } from '../metadata/index.js'
+import { cidOfNode, cidToString } from '../cid/index.js'
+import { decodeIPLDNodeData, IPLDNodeData, OffchainMetadata } from '../metadata/index.js'
 import { Builders, fileBuilders, metadataBuilders } from './builders.js'
 import { createFolderInlinkIpldNode, createFolderIpldNode } from './nodes.js'
 import { chunkBuffer, encodeNode } from './utils.js'
 
 export const DEFAULT_MAX_CHUNK_SIZE = 1024 * 64
 export const DEFAULT_MAX_LINK_PER_NODE = DEFAULT_MAX_CHUNK_SIZE / 64
```

Comment:
It seems weird to first multiply 1024 by 64, then dividing it by 64, Why not switching the order
```
export const DEFAULT_MAX_LINK_PER_NODE = 1024
export const DEFAULT_MAX_CHUNK_SIZE = DEFAULT_MAX_LINK_PER_NODE * 64
```

Reply:
I agree is weird, I updated to be less confusing though I'd prefer to show `DEFAULT_MAX_LINK_PER_NODE` dependant on `DEFAULT_MAX_LINK_PER_NODE` because shows better the logical process that I made for those numbers.